agent/api/task/task.go

```diff
@@ -823,10 +872,12 @@ func (task *Task) addFirelensContainerDependency() error {
 	return nil
 }
 
-// collectLogOptions collects the log options for all the containers that use the firelens container
+// collectFirelensLogOptions collects the log options for all the containers that use the firelens container
 // as the log driver.
-func (task *Task) collectLogOptions() (map[string]map[string]string, error) {
-	containerToLogOptions := make(map[string]map[string]string)
+func (task *Task) collectFirelensLogOptions(containerToLogOptions map[string]map[string]string) error {
```

Comment:
can you add a simple example of this nested map return value -- as I'm reading over this code I get tangled up... actually I get tangled above where this is used `containerToLogOptions := make(map[string]map[string]string)` but I think the comment would be nice here.

Reply:
I see this in the PR detail now.  That's good enough.